src/stories/ProfileHeader.stories.tsx

```diff
@@ -78,17 +83,82 @@ export const Example: Story = {
     const [profileHeaderPopupOpen, setProfileHeaderPopupOpen] = useState(
       args.profileHeaderPopupOpen
     );
+    const [tabKeyPressedCount, setTabKeyPressedCount] = useState(0);
+
+    const ref = useRef<HTMLDivElement>(null);
+
+    const handleClickOutsidePopup = (ev: any) => {
```

Comment:
I can't seem to find a way around utilizing `any` here and on line 101. Any ideas?

Reply:
Let's discuss this and the `handleKeyDown` function a bit more.  I expected them to be part of the component itself because we don't want to require users of this component to have to implement this functionality themselves.  We want to try and build it into the component.